### PR TITLE
[CLI] fix: codemod login experience is broken when user is already logged in - CDMD-2770

### PR DESCRIPTION
**Pull request by hbjORbj**

### CODE REVIEW

File: apps/cli/src/handleLoginCliCommand.ts [hunk 1]
```diff
@@ -7,27 +8,54 @@ import { boldText, colorizeText, openURL } from "./utils.js";
 
 const ACCESS_TOKEN_REQUESTED_BY_CLI_KEY = "accessTokenRequestedByCLI";
 
+const routeUserToStudioForLogin = (printer: PrinterBlueprint) => {
+	printer.printConsoleMessage(
+		"info",
+		colorizeText(
+			"Opening the Codemod Studio... Please Sign in there!\n",
```
Comment (alexbit-codemod) [inline apps/cli/src/handleLoginCliCommand.ts:15]: Avoid studio. We need to change that soon. 

"Redirecting to Codemod sign-in page..."

@hbjORbj

File: apps/cli/src/handleLoginCliCommand.ts [hunk 2]
```diff
@@ -7,27 +8,54 @@ import { boldText, colorizeText, openURL } from "./utils.js";
...
+			"cyan",
+		),
+	);
+	const success = openURL(
+		`https://codemod.studio/?command=${ACCESS_TOKEN_REQUESTED_BY_CLI_KEY}`,
+	);
+	if (!success) {
+		printer.printOperationMessage({
+			kind: "error",
+			message: "Unexpected error occurred while opening the Codemod Studio.",
```
Comment (alexbit-codemod) [inline apps/cli/src/handleLoginCliCommand.ts:25]: Avoid the term studio. 
"Unexpected error occurred while redirecting to sign-in page. Please submit a GitHub issue (sub.co shortlink to github issue template, so we can change it later via dub) or report to codemod.com/community"

**Review by alexbit-codemod — COMMENTED** [review]
Left some comments. Good to have. Not a blocker.

File: apps/cli/src/handleLoginCliCommand.ts [hunk 3]
```diff
@@ -7,27 +8,54 @@ import { boldText, colorizeText, openURL } from "./utils.js";
...
+		if (username === null) {
+			routeUserToStudioForLogin(printer);
+			return;
+		}
+
 		printer.printConsoleMessage(
 			"info",
 			colorizeText(
-				"Opening the Codemod Studio... Please Sign in with Github!\n",
+				boldText("You are already logged in with the Codemod CLI!"),
```
Comment (alexbit-codemod) [inline apps/cli/src/handleLoginCliCommand.ts:55]: "You're already logged in."

File: apps/cli/src/handleLoginCliCommand.ts (hunk 2 above)
Comment (hbjORbj) [reply apps/cli/src/handleLoginCliCommand.ts]: i think devs will feel more safe knowing that the URL links to github, so I will just leave it as native link

**Review by DmytroHryshyn — APPROVED** [review]

**Pull request merged**